packages/config/src/layer2s/arbitrum.ts

```diff
@@ -347,108 +342,62 @@ export const arbitrum: Layer2 = {
-        description:
-          'Contract managing Inboxes and Outboxes. It escrows ETH sent to L2.',
-        upgradeability: discovery.getContract(
-          '0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a',
-        ).upgradeability,
-      },
-      {
-        address: EthereumAddress(
-          discovery.getContractValue<string>('RollupProxy', 'outbox'),
-        ),
-        name: 'Outbox',
-        upgradeability: discovery.getContract(
-          discovery.getContractValue<string>('RollupProxy', 'outbox'),
-        ).upgradeability,
       },
       {
-        address: EthereumAddress('0x9aD46fac0Cf7f790E5be05A0F15223935A0c0aDa'),
-        name: 'ProxyAdmin (3)',
-        description:
-          'This is yet another proxy admin for the three gateway contracts below. It is owned by the Upgrade Executor.',
+        ...discovery.getContractFromValue('RollupProxy', 'outbox'),
+        // TODO: add description
       },
       {
-        address: discovery.getContract('L1GatewayRouter').address,
-        name: 'L1GatewayRouter',
+        ...discovery.getMainContractDetails('L1GatewayRouter'),
         description: 'Router managing token <--> gateway mapping.',
-        upgradeability: discovery.getContract('L1GatewayRouter').upgradeability,
       },
       {
-        address: discovery.getContract('L1ERC20Gateway').address,
-        name: 'L1ERC20Gateway',
+        ...discovery.getMainContractDetails('L1ERC20Gateway'),
         description:
           'Main entry point for users depositing ERC20 tokens. Upon depositing, on L2 a generic, "wrapped" token will be minted.',
-        upgradeability: discovery.getContract('L1ERC20Gateway').upgradeability,
       },
       {
-        address: discovery.getContract('L1CustomGateway').address,
-        name: 'L1CustomGateway',
+        ...discovery.getMainContractDetails('L1CustomGateway'),
         description:
           'Main entry point for users depositing ERC20 tokens that require minting custom token on L2.',
-        upgradeability: discovery.getContract('L1CustomGateway').upgradeability,
       },
       {
-        address: EthereumAddress('0xD3B5b60020504bc3489D6949d545893982BA3011'),
-        name: 'L1DaiGateway',
+        ...discovery.getMainContractDetails('L1DaiGateway'),
         description:
           'Custom DAI Gateway, main entry point for users depositing DAI to L2 where "canonical" L2 DAI token managed by MakerDAO will be minted. Managed by MakerDAO.',
       },
       {
-        address: EthereumAddress('0xA10c7CE4b876998858b1a9E12b10092229539400'),
-        name: 'L1Escrow',
+        ...discovery.getMainContractDetails('L1DaiEscrow'),
         description: 'DAI Vault for custom DAI Gateway managed by MakerDAO.',
       },
+      {
+        ...discovery.getMainContractDetails('UpgradeExecutor'),
+        description:
+          "This contract can upgrade the system's contracts. The upgrades can be done either by the Security Council or by the L1ArbitrumTimelock.",
+      },
```

Comment:
Ok, either we include the description in the helper function or we don't. Either works imo